lsp/src/saros/lsp/extensions/server/account/AccountService.java

```diff
@@ -1,15 +1,118 @@
 package saros.lsp.extensions.server.account;
 
+import java.util.List;
 import java.util.concurrent.CompletableFuture;
+import saros.account.XMPPAccount;
+import saros.account.XMPPAccountStore;
+import saros.lsp.extensions.server.SarosResponse;
+import saros.lsp.extensions.server.SarosResultResponse;
+import saros.lsp.extensions.server.account.dto.AccountDto;
+import saros.lsp.extensions.server.account.dto.AddInput;
+import saros.lsp.extensions.server.account.dto.RemoveInput;
+import saros.lsp.extensions.server.account.dto.SetActiveInput;
+import saros.lsp.extensions.server.account.dto.UpdateInput;
+import saros.net.xmpp.JID;
 
 /** Implementation of the account service. */
 public class AccountService implements IAccountService {
 
+  private final XMPPAccountStore accountStore;
+
+  public AccountService(final XMPPAccountStore accountStore) {
+    this.accountStore = accountStore;
+  }
+
+  @Override()
+  public CompletableFuture<SarosResultResponse<AccountDto[]>> getAll() {
```

Comment:
Are there ANY ?! plans to make these methods async in the future ? If not then please drop this CompleteableFuture stuff.